release-notes.md

```diff
@@ -10,11 +10,28 @@ Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r
 
 ##  March 16, 2020
 
+The following updates were made in this release:
+
+### iOS Core 2.5.1
+
+* Fixed an crash which happened on `ADBJsonType::Get`.
+* Fixed an crash which happened on `EventHubInner::CreateOrUpdateSharedStateCommon`, the crash was introduced on the version 2.5.0.
```

Comment:
Fixed a crash which happened in `EventHubInner::CreateOrUpdateSharedStateCommon`, the crash was introduced in version 2.5.0.